language-server/modules/langserver-core/src/test/resources/completion/new/function_def/config/config15.json

```diff
@@ -1,507 +1 @@
-{
```

Comment:
This test case has been broken by the change

Reply:
Create a separate issue for this(https://github.com/ballerina-platform/ballerina-lang/issues/28946)